pkg/controllers/configobservation/console/observe_consoleurl_test.go

```diff
@@ -120,7 +126,7 @@ func TestObserveConsoleURL(t *testing.T) {
 			}
 
 			for i := range errs {
-				if strings.Contains(tt.expectedErrs[i], errs[i].Error()) {
+				if !strings.Contains(errs[i].Error(), tt.expectedErrs[i]) {
```

Comment:
this logic was wrong in two ways.

as the tests are written, the expected error string is a subset of the actual error string.

and of course the test should fail if an error occurs that was not in the expected list, not fail when an expected error occurs.